data_model/src/lib.rs

```diff
@@ -811,10 +811,10 @@ pub mod model {
         /// Transaction hash
         Transaction(HashOf<VersionedSignedTransaction>),
         /// Block hash
-        Block(HashOf<VersionedCommittedBlock>),
+        Block(HashOf<VersionedSignedBlock>),
     }
 
-    /// Cross-platform wrapper for [`VersionedCommittedBlock`].
+    /// Cross-platform wrapper for [`VersionedSignedBlock`].
```

Comment:
it is not a workaround for a bug. AFAIK it is because it fires `large_enum_variant` on M1 but not on x86. Can't remember the exact reason for this though